### ISSUE TITLE
Using moar literals

### ISSUE THREAD

**Issue by alfonsogarciacaro** [MangelMaxime/Fable.Elmish.Bulma#19]
In the BulmaClasses.fs file there are many record definitions, but in most cases the purpose is only to expose a series of literal values. For example:

```fsharp
let button : Button =
 { Container = "button"
   Size =
      { IsSmall = "is-small"
        IsMedium = "is-medium"
        IsLarge = "is-large "
        IsFullwidth = "is-fullwidth" }
    ... }
```

Using modules with literal values instead, we can get the same API (so there's no need to change client code) but the string values will be written directly in the generated JS code, which should be cleaner and more performant:

```fsharp
module button =
   let [<Literal>] Container = "button"
   module Size =
      let [<Literal>] IsSmall = "is-small"
      let [<Literal>] IsMedium= "is-medium"
      ...
```

**Comment by kunjee17** [comment]
@MangelMaxime should I take this for current code?

**Comment by MangelMaxime** [comment]
@kunjee17 If you want to fix this issue yes you can go from the master branch.
Do you want to assign you on this ?

**Comment by kunjee17** [comment]
@MangelMaxime even if you don't assign. It is ok. I ll go from Master branch. Thanks :)

**Comment by MangelMaxime** [comment]
Will be fixed in 0.1.0-alpha8
